src/com/dotmarketing/util/Config.java

```diff
@@ -329,5 +338,25 @@ public static void forceRefresh(){
 
 	}
 
+	public static void _setupFakeTestingContext() throws Exception{
```

Comment:
Creates a mock fake ServletContext.  Had to create a mock for the context.getRealPath which we use all over.